### ISSUE TITLE
Custom Linter .lesshint file require throws error

### ISSUE THREAD

**Issue by justtal** [lesshint/lesshint#329]
**Which version of `lesshint` are you using?** 

2.4.0

**How are you running `lesshint`? CLI, Node.js API, Grunt/Gulp plugin?**

CLI

**What's your `.lesshintrc` configuration?**

```js
{
  "linters": [
      require("./linters/custom-linter")
  ],
  "singleLinePerSelector": false,
  "propertyOrdering": false,
  "importantRule": false,
  "zeroUnit": false,
  "decimalZero": false,
  "newlineAfterBlock": false,
  "hexNotation": false,
  "qualifyingElement": false,
  "spaceBeforeBrace": {
     "style": "one_space"
  },
  "singleLinePerProperty": true,
  "hexLength": false,
  "spaceAroundOperator": false,
  "attributeQuotes": false,
  "stringQuotes": false,
  "finalNewline": false,
  "idSelector": false,
  "borderZero": false,
  "maxCharPerLine": false,
  "emptyRule": true,
  "trailingSemicolon": true,
  "spaceAfterPropertyColon": {
    "style": "one_space"
  }
}
```
**If you're reporting a bug, please show us some code that's failing.**

```js
 "linters": [
      require("./linters/custom-linter")
  ],
```

**What did you expect to happen?**

Run the custom linter

**What's the actual result?**

Something's wrong with the config file. Error: Unexpected token r

**Would you be interested in submitting a PR for this issue?**

Didn't manage to diving into lesshint code

**Comment by shellscape** [comment]
(updated; please use backticks for code)

**Comment by justtal** [comment]
No problem!

Thanks for the fix.

**Comment by jwilsson** [comment]
This looks like a docs issue, sorry about that!

Since `.lesshintrc` is JSON, `require` calls aren't possible. You'll need to specify the full path to your linter, i.e.

```json
"linters": [
  "/full/path/to/my-custom-linter.js"
]
```

I'll get the docs updated.

**Comment by justtal** [comment]
Thanks.

Yes I looks wired to me to put require in json attribute.

Now I get:

`Cannot find module '/Users/username/dev/some/path/to/custom-linter.js'`

I'm using MAC.

**Comment by justtal** [comment]
There is no option to specify a relative to config file linter path?

It is really problem to do it full path if we work as a team in a different machines and also it will run on jenkins machines :)

Can I try to solve that problem and push a request if I'll succeed?

**Comment by shellscape** [comment]
They just merged a PR today to assist with that https://github.com/lesshint/lesshint/pull/328

**Comment by justtal** [comment]
Works great.

Are you going to release it soon with a new version via npm?

**Comment by jwilsson** [comment]
Yes, it's due for our `3.0` release which we'll hopefully be finished with really soon.

**Comment by jwilsson** [comment]
Closing this since it's fixed in `3.0.0`.
